BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/SelfDiagnostics/SelfDiagnosticsConfigParser.cs

```diff
@@ -40,8 +43,14 @@ public bool TryGetConfiguration(out string logDirectory, out int fileSizeInKB, o
             {
                 var configFilePath = ConfigFileName;
 
+                if (PlatformSingleton.Current.TryGetEnvironmentVariable(LogDiagnosticsEnviornmentVariable, out string logDiagnosticsPath))
+                {
+                    configFilePath = Path.Combine(logDiagnosticsPath, ConfigFileName);
+                    logDirectory = logDiagnosticsPath;
+                }
+
                 // First check using current working directory
```

Comment:
```suggestion
                // Second check using current working directory
```

Update the comments for all three checks accordingly.